nservicebus/security/encryption.md

```diff
@@ -43,7 +43,40 @@ You can also use the `WireEncryptedString` type to flag that a property should b
 
 ### Enabling property encryption
 
-Property encryption is enabled via the configuration API.
+Property encryption is enabled via the configuration API. 
+
+#### Key identifier
+
+Each key needs an unique key identifier (`KeyIdentifier`). The key identifier is communicated in the message header meta data and provides the receiving endpoint information on which key to use for decryption.
+
+NOTE: If a key identifier is not set then no messages can be send but received messages without a key identifier header will be decrypted using the keys that have no or an empty KeyIdentifier set.
```

Comment:
What if you have endpoints A and B. You deploy the fix to both but don't change configs.

At this point, does everything run the same?

Now you update the config on A to include the key identifier on the existing key. It starts sending messages out. Even though B has the correct key, it won't be attempted, and the message can therefore not be processed?